### ISSUE TITLE
No longer delete files when copying "other" files to target folders.

### ISSUE THREAD

**Issue by jeffschwartz** [4awpawz/trio#126]
This behaviour (*implemented in `lib/generator/copyOtherFiles`*) interferes with `package.json npm scripts` that might contribute content to the `public folder` prior to trio running its generation process (e.g. JavaScript bundling libraries such as Parcel and Browserify). This would, then, delete those files that those npm scripts had contributed.

The reasoning to maintain the public folder by removing stale content (i.e. content no longer being generated, such as content that has been deleted or renamed for example) was admirable but really not necessary because *a similar result can be achieved by the user by merely running a one off build (i.e. `trio b`)*.

The bottom line is that cruft buildup in the `public folder` won't impact testing because there are no references to these assets anymore and that is why they can exists side by side with valid current project assets which are referenced.
